docs/compiling-mono/mac/index.md

```diff
@@ -43,6 +43,12 @@ for i in *.tar.gz; do tar xzvf $i; done
 for i in */configure; do (cd `dirname $i`; ./configure --prefix=$PREFIX && make && make install); done
 ```
 
+Alternatively, dependencies can be installed through Homebrew:
+
+``` bash
+brew install autoconf automake libtool pkg-config glib libmcrypt freetype cmake libmemcached
```

Comment:
@akoeplinger - I kept instaling dependencies until the build succeeded. Without libmemcached I got an error saying it couldn't find `libmemcached/aclocal.m4`. However I just tried it again and it worked this time even after uninstalling libmemcached so I'm not sure why it complained initially.

@Therzok `brew deps mono` gives me an empty result.